### PR TITLE
if: add page

### PR DESCRIPTION
**Pull request by EmilySeville7cfg**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

For #7440

### CODE REVIEW

File: pages/windows/if.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# if
+
+> Performs conditional processing in batch programs.
+> More information: <https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/if>.
```
Comment (navarroaxel) [inline pages/windows/if.md:4]: We don't use the `i18n` parameter for Microsoft URLs
```suggestion
> More information: <https://docs.microsoft.com/windows-server/administration/windows-commands/if>.
```

File: pages/windows/if.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+- Check whether two strings are equal without respecting letter case:
+
+`if /i {{string}} == {{string}} {{command}}`
+
+- Check whether file exist:
+
+`if exist {{path/to/file}} {{command}}`
+
+- Check whether negated condition is true:
```
Comment (navarroaxel) [inline pages/windows/if.md:22]: Parhaps this better
```suggestion
- Check whether a condition is false:
```

File: pages/windows/if.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+- Check whether two strings are equal:
+
+`if {{string}} == {{string}} {{command}}`
+
+- Check whether two strings are equal without respecting letter case:
+
+`if /i {{string}} == {{string}} {{command}}`
+
+- Check whether file exist:
```
Comment (navarroaxel) [inline pages/windows/if.md:18]:
```suggestion
- Check whether a file exist:
```

File: pages/windows/if.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
+# if
+
+> Performs conditional processing in batch programs.
+> More information: <https://docs.microsoft.com/windows-server/administration/windows-commands/if>.
+
+- Check whether `%errorlevel%` is greater than or equal to specified value:
+
+`if errorlevel {{exit_code}} {{command}}`
```
Comment (marchersimon) [inline pages/windows/if.md:8]: Is `exit_code` the specified value you're comparing too?

File: pages/windows/if.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
+# if
+
+> Performs conditional processing in batch programs.
+> More information: <https://docs.microsoft.com/windows-server/administration/windows-commands/if>.
+
```
Comment (marchersimon) [inline pages/windows/if.md:5]: I'd add a more general example here to explain what `{{command}}` is.
```suggestion

- Execute a command, if a specific condition is true:

`if {{condition}} {{command}}`

```

File: pages/windows/if.md (hunk 4 above)
Comment (EmilySeville7cfg) [reply pages/windows/if.md:8]: Yes.

Comment (marchersimon) [reply pages/windows/if.md:8]: Then we sholud probably change the description to
``- Check whether `%errorlevel%` is greater than or equal to the specified exit code:``

File: pages/windows/if.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+`if {{condition}} ({{commands}})`
+
+- Execute the specified commands if the condition is false:
+
+`if not {{condition}} ({{commands}})`
+
+- Execute the first specified commands if the condition is true otherwise execute the second specified commands:
+
+`if {{condition}} ({{first_commands}}) else ({{second_commands}})`
```
Comment (navarroaxel) [inline pages/windows/if.md:16]: Why we don't use the same command examples than the [common/if.md](https://github.com/tldr-pages/tldr/blob/main/pages/common/if.md) page? sounds easier to read and understand.
```suggestion
`if {{condition}} ({{echo "true"}}) else ({{echo "false"}})`
```

Comment (EmilySeville7cfg) [reply pages/windows/if.md]: Good suggestion. ;) I've replaced almost all occurrences of placeholders but preserved general descriptions.

File: pages/windows/if.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
+# if
+
+> Performs conditional processing in batch scripts.
+> More information: <https://docs.microsoft.com/windows-server/administration/windows-commands/if>.
+
+- Execute the specified commands if the condition is true:
+
+`if {{condition}} (echo Condition is true)`
```
Comment (sbrl) [inline pages/windows/if.md:8]: Should the echo command use double quotes to surround the thing that is being echoed?

Comment (EmilySeville7cfg) [reply pages/windows/if.md:8]: No, because it treat them literally.

Comment (sbrl) [inline pages/windows/if.md:8]: Oh yeah, we're missing some token syntax here too - the same also goes for the other examples.
```suggestion
`if {{condition}} ({{echo Condition is true}})`
```

Comment (neowinston) [reply pages/windows/if.md:8]: Agreed.

Comment (EmilySeville7cfg) [reply pages/windows/if.md:8]: But `echo Condition is true` is not a placeholder, it's a concrete command. I've removed this placeholder to simplify tlds page.

**Review by sbrl — APPROVED** [review]
Thanks, @EmilySeville7cfg!

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**